plumbing/msg/previewer.go

```diff
@@ -48,9 +48,13 @@ func (p *Previewer) Preview(ctx context.Context, optFrom, to address.Address, me
 	if err != nil {
 		return types.NewGasUnits(0), errors.Wrap(err, "could load tree for latest state root")
 	}
-	h, err := p.chainReader.BlockHeight()
+	head, err := p.chainReader.GetTipSet(p.chainReader.GetHead())
 	if err != nil {
-		return types.NewGasUnits(0), errors.Wrap(err, "couldnt get base tipset height")
+		return types.NewGasUnits(0), errors.Wrap(err, "couldnt get head tipset ")
```

Comment:
nit: here and in `plumbing/msg/queryer.go`, lets's make these error message more consistent -- "failed to get head tipset"